src/utils/service.py

```diff
@@ -0,0 +1,216 @@
+"""Utility module containing logic of creation and deletion of managed Kyuubi K8s service."""
```

Comment:
**suggestion**:  What about making this a manager to fit the pattern a bit more? I try (though unsuccessfully sometimes!) to avoid `utils`, which tends to evolve into a dumping ground for unrelated stuff

Reply:
I agree. My initial thoughts of making this an util came from an assumption that `utils.service` will be moved to `data-platform-helpers` and us having to change only `from utils.service` to `from data-platform-helpers.service` and the rest being exactly the same.

Reply:
I've refactored to put this logic in a manager instead of a util.